### ISSUE TITLE
Shifted end-of-day mode enabled but not functioning

### ISSUE THREAD

**Issue by GwyndolynMarchant** [Hoffelhas/automation-todoist#13]
### Setup:
- Ubuntu 18.04.5 LTS
- Python 3.6.9
  - todoist-python 8.1.3
  - requests 2.25.1

Running autodoist with the following settings: `python3 autodoist.py -a <####> -l next_action -e 7`
Have also attempted: `python3 autodoist.py -a=<####> -l=next_action -e=7`

Autodoist reports:
```
INFO     You are running with the following functionalities:

   Next action labelling mode: Enabled
   Regenerate sub-tasks mode: Disabled
   Shifted end-of-day mode: Enabled
```

### Expected result:
When checking off an overdue daily task before 7am (07:00), Todoist normally sets the task to now have a due date of `tomorrow`. Autodoist should, after refreshing, update this due date to `today`.

### What happens:
Autodoist does not update the due date.

**Comment by GwyndolynMarchant** [comment]
Confirmed problem running from the following system configuration as well:

- Windows 10 (19042)
- Python 3.9.4
  - todoist-python 8.1.3
  - requests 2.25.1

**Comment by GwyndolynMarchant** [comment]
Proposed fix in #15

**Comment by Hoffelhas** [comment]
Hi, sorry it took a while. Personal life has been quite busy this year.

In any case, nice catch! The bug was that a reference date was updated with the current date, before it checked the offset with the current date. Hence, part of the logic wouldn't trigger. Your change indeed fixed that, so thanks for the help!

For now I'll close this topic, if anything new arises, just be sure to reach out again.
